### ISSUE TITLE
Preferences - Click on option's text should select/tick option

### ISSUE THREAD

**Issue by Eagle3386** [fnxweb/urllink#19]
Selecting an option, e. g. "Prefer new window", or ticking it, e. g. "Hide the normal 'open' options", requires to _exactly_ hit the check box/radio button.

This could (and IMHO should) be more user-friendly by extending the hit target to an option's label - most websites I visit and even Firefox itself do so already.

**Comment by fnxweb** [comment]
Good point. I may even find some time to get some fixes in one day (my life is currently full of post-house-move DIY).
